pyoverkiz/types.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+import json
+from typing import Any, Callable, Dict, List, Union
+
+from pyoverkiz.enums import DataType
+
+OverkizStateType = Union[str, int, float, bool, Dict[Any, Any], List[Any], None]
+
+DATA_TYPE_TO_PYTHON: dict[DataType, Callable[[Any], OverkizStateType]] = {
+    DataType.INTEGER: int,
+    DataType.DATE: int,
+    DataType.STRING: str,
+    DataType.FLOAT: float,
+    DataType.BOOLEAN: bool,
+    DataType.JSON_ARRAY: json.loads,
+    DataType.JSON_OBJECT: json.loads,
```

Comment:
It will avoid the special case in your init.
```suggestion
    DataType.JSON_OBJECT: json.loads,
    DataType.NONE: lambda x: x
```

Reply:
```
pyoverkiz/types.py:11: error: Returning Any from function declared to return "Union[str, int, float, bool, Dict[Any, Any], List[Any], None]"  [no-any-return]
Found 1 error in 1 file (checked 2 source files)
```

Reply:
It seems that this is not possible, I will need to have a look.

Reply:
I guess we have to type the lambda

Reply:
I just did :D